tests/sentry/rules/processing/test_delayed_processing.py

```diff
@@ -733,6 +733,40 @@ def test_apply_delayed_percent_comparison_condition_interval(self):
         assert (percent_comparison_rule.id, group5.id) in rule_fire_histories
         self.assert_buffer_cleared(project_id=self.project.id)
 
+    def test_apply_delayed_event_frequency_percent_comparison_interval(self):
```

Comment:
I'm not creating any sessions for this test b/c I don't want to trigger this condition, but rather that it's just using the right comparison_interval mapping. It fails with a KeyError without the fix